src/com/lftechnology/java/training/alina/jdbc/service/UtilityService.java

```diff
@@ -0,0 +1,126 @@
+package com.lftechnology.java.training.alina.jdbc.service;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.alina.jdbc.constants.Constants;
+import com.lftechnology.java.training.alina.jdbc.dao.employee.impl.EmployeeDaoImpl;
+import com.lftechnology.java.training.alina.jdbc.domain.Employee;
+
+/**
+ * Utility service consists of common functionalities
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class UtilityService {
+
+    private static final Logger LOGGER = Logger.getLogger(UtilityService.class.getName());
+
+    private UtilityService() {
+
+    }
+
+    /**
+     * Gets input data,validates empty field
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @param fieldLabel
+     *            {@link String}
+     * @return {@link String} input value
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public static String getInputData(Scanner scanner, String fieldLabel) {
+        String fieldValue = null;
+        boolean emptyStatus = true;
+        while (emptyStatus) {
+            LOGGER.log(Level.INFO, fieldLabel);
+            if (!scanner.hasNextLine()) {
+                scanner.nextLine();
+            } else {
+                fieldValue = scanner.nextLine();
+                if (fieldValue.isEmpty()) {
+                    boolean displayStatus = checkEmptyField(fieldLabel);
+                    if (!displayStatus) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS134)